src/content/blog/astro-5120.mdx

```diff
@@ -95,25 +95,30 @@ Until now, using Netlify primitives with Astro during local development required
 
 After upgrading to the latest version of the adapter, running `astro dev` will provide:
 
-- A local Netlify Image CDN for [Astro images](https://docs.astro.build/en/guides/images/)
-- A local Blobs server for [sessions](https://docs.astro.build/en/guides/sessions/) 
-- Redirects, rewrites](https://docs.netlify.com/routing/redirects/) and [headers](https://docs.netlify.com/routing/headers/#app)  from your Netlify config
+- A local [Netlify Image CDN](https://docs.netlify.com/build/image-cdn/overview/#local-development) server. This is used for [images](#netlify-image-cdn-support) by default
+- A local [Netlify Blobs](https://docs.netlify.com/build/data-and-storage/netlify-blobs/) server. This is used for [sessions](#sessions) by default
+- [Redirects, rewrites](https://docs.netlify.com/routing/redirects/) and [headers](https://docs.netlify.com/routing/headers/#app) from your Netlify config
 - The [Netlify Edge Context](https://docs.astro.build/en/guides/integrations-guide/netlify/#accessing-edge-context-from-your-site) in on-demand pages
+- [Environment variables](https://docs.netlify.com/environment-variables/overview/) populated from your linked Netlify site
 
-### Environment variables
+### Configuring dev features
 
-These features are a zero-config upgrade, requiring no changes to your project.
+Along with these features comes a way to configure them. The Netlify adapter now accepts a `devFeatures` option in its configuration, allowing you to enable or disable specific features during development. You can set this to `true` or `false` to enable or disable all features, or you can configure them individually. Currently the features you can configure are:
```

Comment:
On a more higher level, we use "enable and disable" two times here and kind of repeat the information, maybe using "toggle" and trimming it a bit could work better? Not sure if the way I wrote makes it clear the difference between passing an object for specific features (example below) or all features (passing a boolean directly to `devFeatures`)
```suggestion
Along with these features comes a way to configure them. The Netlify adapter now accepts a `devFeatures` option in its configuration, allowing you to toggle specific or all features (passing a boolean directly) during development. Currently, the features you can configure are:
```